environment.yml

```diff
@@ -0,0 +1,92 @@
+name: rainbow
+channels:
+- pytorch
+- defaults
+dependencies:
+- asn1crypto=0.23.0=py36h782d450_0
```

Comment:
These versions look rather specific, and are definitely Python 3.6, so these can't be general enough for most setups.